From c4e79952b42ebccb669063e053e3d7ce0f88cc22 Mon Sep 17 00:00:00 2001
From: Chris Liddell <chris.liddell@artifex.com>
Date: Wed, 16 Sep 2020 17:33:30 +0100
Subject: [PATCH] Fix compile failure due to missing header.

A user reported (privately) that a pre-release of an upcoming XCode release
fails to compile due to a missing prototype/declation of abs() in the jpegxr
code. It's clearly a simple omission, as several files use abs() and do include
the relevant header.

So adding it here.
---
 jpegxr/r_strip.c | 1 +
 1 file changed, 1 insertion(+)

diff --git jpegxr/r_strip.c.orig jpegxr/r_strip.c
index 062baf0c5..0c0e2df5a 100644
--- jpegxr/r_strip.c.orig
+++ jpegxr/r_strip.c
@@ -46,6 +46,7 @@
 #endif
 
 # include "jxr_priv.h"
+# include <stdlib.h>
 # include <limits.h>
 # include <assert.h>
 # include <math.h>
-- 
2.17.1
